initializers/dcim_interfaces.yml

```diff
@@ -0,0 +1,12 @@
+#- device: router1
```

Comment:
This device is not in the default set, therefore it does not make sense to reference it here. Please ensure that the values are consistent between your examples and the existing examples.